domains/logistics/structure-organization/README.md

```diff
@@ -26,9 +26,9 @@ Jane, an artificial agent, is deployed as a  _carrier_. Jane can communicate wit
 
 | ID | Question in Natural Language | Example |
 |----|------------------------------|---------|
-| q1 | What are the job positions including their department and associated membership that are not being played by anyone in the organization X?      | What are the job positions including their department and associated membership that are not being played by anyone in the FL Logistics organization? `ex:FL_AccountManager,ex:FL_CommercialDepartment,ex:AccountManager_Commercial_Membership`                                                    |
+| q1 | What are the job positions including their department and associated membership that are not being played by anyone in the organization X?      | What are the job positions including their department and associated membership that are not being played by anyone in the FL Logistics organization? `ex:AccountManager,ex:FL_CommercialDepartment,ex:AccountManager_Commercial_Membership`                                                       |
```

Comment:
No, `ex:FL_CommercialDepartment` is correct. Note that `hmas:isMembershipIn` is defined as `sh:qualifiedValueShape [ sh:class ex:CommercialDepartment ] ;`, thus the membership is expecting an instance of the type `ex:CommercialDepartment` instead of the `ex:CommercialDepartment` itself.

Reply:
I see, thank you.